flink-table/flink-sql-jdbc-driver-bundle/pom.xml

```diff
@@ -77,7 +89,9 @@
 									<include>org.apache.flink:flink-sql-jdbc-driver</include>
 									<include>org.apache.flink:flink-sql-client</include>
 									<include>org.apache.flink:flink-sql-gateway-api</include>
+									<include>org.apache.flink:flink-sql-gateway</include>
 									<include>org.apache.flink:flink-table-common</include>
+									<include>org.apache.flink:flink-core</include>
```

Comment:
Finally, we include 'flink-core' module 😁

Reply:
I have created an issue https://issues.apache.org/jira/browse/FLINK-31687 to get rid of flink-core after we complete main functions in jdbc driver